src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -54,11 +56,35 @@ export const hasRequiredMhvAccount = (appId, accountLevel) => {
 export const isHealthTool = appId => HEALTH_TOOLS.includes(appId);
 
 export const mhvBaseUrl = () => {
-  const mhvSubdomain = !environment.isProduction() ? 'mhv-syst' : 'www';
+  const mhvSubdomain = environment.isProduction() ? 'www' : 'mhv-syst';
 
   return `https://${mhvSubdomain}.myhealth.va.gov`;
 };
 
+const mhvToEauthRoutes = {
+  'download-my-data': 'download_my_data',
+  'web/myhealthevet/refill-prescriptions': 'prescription_refill',
+  'secure-messaging': 'secure_messaging',
+  appointments: 'appointments',
+  'labs-tests': 'lab-tests',
+};
+
+const mhvUrlNonSSO = subroute => {
+  const mhvSubdomain = environment.isProduction() ? 'www' : 'mhv-syst';
+
+  return `https://${mhvSubdomain}.myhealth.va.gov/mhv-portal-web/${subroute}`;
```

Comment:
could you just use `mhvBaseUrl()` here and delete line 73?

Reply:
good call

- [ ] utilize `mhvBaseUrl` helper